src/dune/vimpl.ml

```diff
@@ -6,7 +6,7 @@ type t =
   ; vlib_modules : Modules.t
   ; vlib_foreign_objects : Path.t list
   ; impl_cm_kind : Cm_kind.t
-  ; vlib_obj_map : Module.Name_map.t Lazy.t
```

Comment:
This was an example of confusing code. The module names in this map were "mangled" names and it's a mistake to look up unmangled `Module_name.t` values in this map.